GOVERNANCE.md

```diff
@@ -298,6 +311,126 @@ process](https://github.com/crossplane/org/blob/main/processes/new-member.md)
 outlined in the `crossplane/org` repository. New members should be asked to set
 the visibility of their Crossplane organization membership to public.
 
+## Community Extension Projects
+
+The [`crossplane-contrib`](https://github.com/crossplane-contrib) organization
+serves as a vendor neutral home for community collaboration on crossplane
+related projects and extensions. We commonly refer to the repositories in this
+organization as **"community extension projects"** which includes providers,
+functions and other crossplane extensions. As previously stated, this
+organization and all of its repositories, in addition to the `crossplane`
+organization, are under the governance and policies defined in this document.
+They are effectively sub-projects of Crossplane and must follow all the same
+rules and policies.
+
+### Benefits of Becoming a Community Extension Project
+
+It is entirely optional for a project in the greater Crossplane ecosystem to
+formally join the `crossplane-contrib` organization as a community extension
+project. The choice is up to the leadership team of each project. The benefits
+of becoming a community extension project are listed below:
+
+* To have a neutral home for collaboration with strong open source governance
+  and licensing
+* To attract more contributors and maintainers and increase the project’s long
+  term sustainability
+* To make the project eligible for reference in the Crossplane documentation and
+  website
+* To contribute and participate more directly in the Crossplane community
```

Comment:
Is it worth mentioning anything about having access to the xpkg.crossplane.io distribution here?

Reply:
good call, will add that too!